core/src/com/unciv/ui/mapeditor/TileEditorOptionsTable.kt

```diff
@@ -293,13 +293,19 @@ class TileEditorOptionsTable(val mapEditorScreen: MapEditorScreen): Table(Camera
 
     fun updateTileWhenClicked(tileInfo: TileInfo) {
         when {
-            clearTerrainFeature -> tileInfo.terrainFeature = null
+            clearTerrainFeature -> {
+                tileInfo.terrainFeature = null
+                tileInfo.naturalWonder = null
+            }
             clearResource -> tileInfo.resource = null
             selectedResource != null -> tileInfo.resource = selectedResource!!.name
             selectedTerrain != null -> {
                 if (selectedTerrain!!.type == TerrainType.TerrainFeature)
```

Comment:
you can use `when` here, it will be more readable

Reply:
i don't get it where: it is encapsulated in `when`